GVFS/GVFS.UnitTests/Platform.Mac/MacDaemonManagerTests.cs

```diff
@@ -0,0 +1,41 @@
+using GVFS.Common;
+using GVFS.Platform.Mac;
+using GVFS.Tests.Should;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace GVFS.UnitTests.Platform.Mac
+{
+    [TestFixture]
+    public class MacServiceProcessTests
+    {
+        [TestCase]
+        public void CanGetServices()

```

Comment:
Do we need any failure tests like a `ProcessResult` that is an error?

Reply:
Noted for followup.